src/transformState.js

```diff
@@ -4,8 +4,31 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
```

Comment:
It would be better to use `switch/case` instead of `if/else` ([checklist](https://github.com/mate-academy/js_stateful-object/blob/master/checklist.md) p2 + p4)